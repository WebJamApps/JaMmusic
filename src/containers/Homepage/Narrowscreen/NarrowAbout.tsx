import React from 'react';
import Faqs from '../HomepageData';

const NarrowAboutUs = (): JSX.Element => (
  <div className="notWidescreen" style={{ marginTop: '1px' }}>
    <div id="slideshow">
      <img alt="Web Jam Banner" src="../static/imgs/banner1.jpg" />
    </div>
    <div style={{ padding: '40px', marginBottom: '-20px' }}>
      <p style={{ marginTop: '0px', marginBottom: '40px', fontSize: '18px' }}>
        Web Jam LLC helps businesses bring their web technologies into the 21st century! Let us
        help make your website mobile-friendly and grow your business by utilizing the power of online services.
        More information is available in our&nbsp;
        <a href="https://www.dropbox.com/s/bzdqp3hr682y9sw/WebJamLLC_FactSheet.pdf?dl=0" target="_blank" rel="noopener noreferrer">Fact Sheet</a>
        .
      </p>
      <Faqs />
    </div>
  </div>
);

export default NarrowAboutUs;
