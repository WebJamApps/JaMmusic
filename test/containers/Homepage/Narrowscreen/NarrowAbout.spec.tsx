import React from 'react';
import { shallow } from 'enzyme';
import NarrowAbout from '../../../../src/containers/Homepage/Narrowscreen/NarrowAbout';

const wrapper = shallow<typeof NarrowAbout>(<NarrowAbout/>);

describe('NarrowAbout', () => {
  it('Renders the NarrowAbout', () => {
    expect(wrapper.find('div.notWidescreen').exists()).toBe(true);
    expect(wrapper).toMatchSnapshot();
  });
});
