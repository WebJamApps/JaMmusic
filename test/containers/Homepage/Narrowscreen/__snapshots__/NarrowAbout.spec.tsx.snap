// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NarrowAbout Renders the NarrowAbout 1`] = `ShallowWrapper {}`;
